cpu/stm32l1/include/cpu_conf.h

```diff
@@ -72,6 +72,20 @@ extern "C" {
 #define CPU_FLASH_BASE                  FLASH_BASE
 /** @} */
 
+/**
+ * @brief   Flash page configuration
```

Comment:
Should be `@name` and also add a closing line `/** @} */` after the defines.

Reply:
Done.